### PR TITLE
Remove EWC from docs

### PR DESCRIPTION
**Pull request by amanda11**
Update documentation to reflect that EWC is not available after StackStorm 3.2, and that the old enterprise components will be added in future community releases.

Addresses part of #985

### CODE REVIEW

**Comment by amanda11** [comment]
Given the movement on the enterprise components, going to mark this as draft....

File: docs/source/authentication.rst [hunk 1]
```diff
@@ -136,9 +136,13 @@ The following is a list of auth backends for the community edition to help get t
...
 LDAP (Enterprise Edition)
 -------------------------
-|st2|-developed auth backends such as LDAP are only available in |ewc|. For more information on
-|ewc|, please visit https://www.extremenetworks.com/product/workflow-composer/
-The auth backends included with |ewc| are developed, tested, maintained, and supported by Extreme Networks.
+.. note::
+
+  Prior to StackStorm 3.3, Extreme Networks provided a commercial version of the StackStorm automation platform including a LDAP auth backend.
+
+  The EWC components have been gifted to the Linux Foundation, but have not yet been integrated into the StackStorm release.
```
Comment (Sheshagiri) [inline docs/source/authentication.rst:143]:
```suggestion
Extreme Workflow Composer was donated to Linux Foundation, but those components are not yet integrated into the StackStorm release. Please visit https://stackstorm.com/2020/05/27/extreme-networks-donates-ewc-to-linux-foundation/ for more details.
```

File: docs/source/authentication.rst [hunk 2]
```diff
@@ -136,9 +136,13 @@ The following is a list of auth backends for the community edition to help get t
 
 LDAP (Enterprise Edition)
 -------------------------
-|st2|-developed auth backends such as LDAP are only available in |ewc|. For more information on
-|ewc|, please visit https://www.extremenetworks.com/product/workflow-composer/
-The auth backends included with |ewc| are developed, tested, maintained, and supported by Extreme Networks.
+.. note::
+
+  Prior to StackStorm 3.3, Extreme Networks provided a commercial version of the StackStorm automation platform including a LDAP auth backend.
```
Comment (Sheshagiri) [inline docs/source/authentication.rst:141]:
```suggestion
  Prior to StackStorm 3.3, Extreme Networks provided a commercial version(Extreme Workflow Composer) of the StackStorm automation platform including an LDAP auth backend.
```

File: docs/source/install/common/ewc_intro.rst [hunk 3]
```diff
@@ -1,11 +1,6 @@
...
-RBAC and LDAP to |st2|. It is deployed as a set of additional packages on top of |st2|. You will need an active
-|ewc| subscription and a license key to access |ewc| repositories.
+For StackStorm versions earlier than 3.3, Extreme Networks provided a commercial version of the StackStorm automation platform (EWC). EWC added priority
+support, advanced features such as fine-tuned access control (RBAC), LDAP, and Workflow Designer.
 
-To learn more about |ewc| check out
-`Product Page <https://www.extremenetworks.com/product/workflow-composer/>`_.
-To request a quote, or get an evaluation license, contact Sales by filling out the form at
-https://www.extremenetworks.com/contact-sales/.
+The EWC components have been gifted to the Linux Foundation, but have not yet been integrated into the StackStorm release.
```
Comment (Sheshagiri) [inline docs/source/install/common/ewc_intro.rst:4]:
```suggestion
Extreme Workflow Composer was donated to Linux Foundation, but those components are not yet integrated into the StackStorm release. Please visit https://stackstorm.com/2020/05/27/extreme-networks-donates-ewc-to-linux-foundation/ for more details.
```

File: docs/source/rbac.rst [hunk 4]
```diff
@@ -3,9 +3,11 @@ Role Based Access Control
 
 .. note::
 
-   Role Based Access Control (RBAC) is only available in |ewc|. For information
-   about |ewc| and the differences between StackStorm and |ewc|, please see
-   `stackstorm.com/product <https://stackstorm.com/product/#enterprise>`_.
+  Prior to StackStorm 3.3, Extreme Networks provided a commercial version of the StackStorm automation platform which included Role Based Access Control (RBAC).
+
+  The EWC components have been gifted to the Linux Foundation, but have not yet been integrated into the StackStorm release.
```
Comment (Sheshagiri) [inline docs/source/rbac.rst:8]:
```suggestion
Extreme Workflow Composer was donated to Linux Foundation, but those components are not yet integrated into the StackStorm release. Please visit https://stackstorm.com/2020/05/27/extreme-networks-donates-ewc-to-linux-foundation/ for more details.
```

**Review by Sheshagiri — CHANGES_REQUESTED** [review]
@amanda11 I just rephrased what you already wrote and added a link for reference.

**Review by Sheshagiri — APPROVED** [review]
lgtm 👍

**Review by armab — APPROVED** [review]
Thanks @amanda11 and @Sheshagiri!

Looks like we'll need to shelf and revert some of these documentation changes once we fully integrate the enterprise bits in the future st2 releases.

**Comment by amanda11** [comment]
> Thanks @amanda11 and @Sheshagiri!
> 
> 
> 
> Looks like we'll need to shelf and revert some of these documentation changes once we fully integrate the enterprise bits in the future st2 releases.

Yes - should be easy to do. I tried to do it so it would be easy to re-instate.

**Pull request merged**